.github/workflows/lintAndTest.yml

```diff
@@ -0,0 +1,30 @@
+name: php cs fixer
+run-name: ${{ github.actor }}
+on:
+  push:
+    branches:
+      - main

```

Comment:
Use `master` (old repo is using master branch)
@ErlendMariusOmmundsen 
@juliadai8